src/util/Parameters.h

```diff
@@ -99,6 +122,21 @@ struct Parameter : public ParameterBase {
   }
 };
 
+// Concept for `Parameter`.
```

Comment:
```suggestion
// Concept that checks whether a type is an instantiation of the `Parameter` template.
// Note: We cannot use `ad_utility::IsInstantiation`, because `Parameter` has a non-type template parameter.
```

Reply:
Done in [679e6aa](https://github.com/ad-freiburg/qlever/pull/1067/commits/679e6aad9d974c4f6d2986f3a294d4edccc88090).